src/main/kotlin/com/terraformation/backend/accelerator/event/ParticipantProjectSpeciesAddedToProjectEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.terraformation.backend.accelerator.event
+
+import com.terraformation.backend.db.accelerator.DeliverableId
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.SpeciesId
+
+/** Published when a participant project species is added to a project */
+data class ParticipantProjectSpeciesAddedToProjectEvent(
```

Comment:
Reading this whole PR, I see the distinction between this event and the other one, but in isolation, they're kind of confusing. `ParticipantProjectSpeciesAddedEvent` also includes a project ID so the comment here would be true of it as well.

Maybe it'd be clearer to rename this to something that specifically mentions "notification due" so people don't think they can listen for this when they really want the other one.

Reply:
Good call, I was hoping making the name a bit more specific would indicate the notification intended, but you're right, it isn't obvious.